docs/README

```diff
@@ -0,0 +1,29 @@
+Steps to build and use the Sphinx documentation tool:
+
+1) Get Sphinx and sphinxcontrib-bibtex installed on your desktop from
+    http://www.sphinx-doc.org/en/master/usage/installation.html
+    https://sphinxcontrib-bibtex.readthedocs.io/en/latest/quickstart.html#installation
+
+2) Create a Sphinx documentation root directory:
+   % mkdir docs
+   % cd docs
+
+3) Initialize your Sphinx project (set up an initial directory structure) using
+   % sphinx-quickstart
```

Comment:
Ah, that file is automatically generated, and I believe the steps you've commented are irrelevant because the project directories are already built. You should just be able to log into a readthedocs account and "Import a Project." If you're looking to build locally, you'd run "make clean && sphinx-build -b html source build". You may have to import Sphinx, but I suspect it'll say which packages are missing, if any. Regardless, I eliminated those confusing lines and could probably just delete the file or add a reference to the Sphinx documentation depending on what you prefer.

Reply:
What I would recommend is just building the docs directly in Read the Docs website and including a link to them in the hpc-stack README. Then no one needs to build locally (and if they choose to anyway, I feel like that's on them, but feel free to disagree).

Reply:
Yes, that's fine. I wouldn't expect many people to generate the docs themselves. Just wanted to try it myself.